backend/src/resolvers/socialMedia.js

```diff
@@ -15,6 +18,11 @@ export default {
       )
       session.close()
 
+      return socialMedia
+    },
+    DeleteSocialMedia: async (object, params, context, resolveInfo) => {
+      const socialMedia = await neo4jgraphql(object, params, context, resolveInfo, false)
```

Comment:
doesn't this create another `SocialMedia` node @Tirokk, or is there some magic that sees that you have called a delete action?

Reply:
There seems to be a magic that deletes the node and with it the relation. @mattwr18 
Because it’s an autogenerated mutation.

Reply:
@mattwr18 you could inspect the generated cypher query of `neo4jgraphql` to get evidence, but I assume it's using `DELETE DETACH`. https://www.graphgrid.com/understanding-detach-delete-in-cypher/

Quite a final solution for a node and it's relationship. Comparable to Rails' `:dependent => :destroy`.